bin/deploy-wmagent.sh

```diff
@@ -230,9 +230,11 @@ cd -
 fi
 echo "Done!" && echo
 
-### Enabling couch watchdog:
+### Enabling couch watchdog; couchdb fix for file descriptors
 echo "*** Enabling couch watchdog ***"
 sed -i "s+RESPAWN_TIMEOUT=0+RESPAWN_TIMEOUT=5+" $CURRENT/sw*/$WMA_ARCH/external/couchdb*/*/bin/couchdb
+sed -i "s+exec 1>&-+exec 1>$CURRENT/install/couchdb/logs/stdout.log+" $CURRENT/sw*/$WMA_ARCH/external/couchdb*/*/bin/couchdb
+sed -i "s+exec 2>&-+exec 2>$CURRENT/install/couchdb/logs/stderr.log+" $CURRENT/sw*/$WMA_ARCH/external/couchdb*/*/bin/couchdb
```

Comment:
Hi Alan, since we will be deploying agent after cmsweb deployment.
I think it is ok to change it now.

```
exec 1>/tmp/couch-stdout.log
exec 2>/tmp/couch-stderr.log
```